hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListener.java

```diff
@@ -287,32 +292,48 @@ private void processNonFeeTransfers(long consensusTimestamp, RecordItem recordIt
         }
 
         var body = recordItem.getTransactionBody();
+        var partialDataAction = parserProperties.getPartialDataAction();
         var transactionRecord = recordItem.getRecord();
         for (var aa : nonFeeTransfersExtractor.extractNonFeeTransfers(body, transactionRecord)) {
             if (aa.getAmount() != 0) {
-                EntityId entityId = getAccountId(aa.getAccountID());
+                EntityId entityId = EntityId.EMPTY;
+                try {
+                    entityId = getAccountId(aa.getAccountID());
+                } catch (AliasNotFoundException ex) {
+                    switch (partialDataAction) {
+                        case DEFAULT:
+                            log.warn(ex.getMessage() + ", action " + partialDataAction);
```

Comment:
nit: Would a regular English sentence here and for skip:
```suggestion
                            log.warn("Setting non-fee transfer account to default value due to partial data issue: {}", ex.getMessage());
```